### ISSUE TITLE
Classifying whole image without segmentation

### ISSUE THREAD

**Issue by hkuyam008** [Labelbox/Labelbox#126]
This is more of a question at this stage than an issue to report.

If I just want to classify / assign category to a set of images (WITHOUT segmentation), ...let's say for example I have a set of car images (with all white background) and I just want to categorize / classify them with labels Honda, Subaru, Tesla etc (without annotation, since there are no background images to segment). shouldn't the generated json data of labelbox include those labels?

Right now the json that I am exporting in WKT/json format has the "Label" element value as {}, which basically means no labels??

I am attaching the generated json for reference - I labelled (selected category) 4 images as "button" / "dropdown" (2 in each category/class) and then exported the resultant json (WKT)

[labelbox_gui_components.zip](https://github.com/Labelbox/Labelbox/files/2387600/labelbox_gui_components.zip)

**Comment by feynmanliang** [comment]
You're absolutely right! This is a bug in our WKT export format; we're only exporting the polygon geometry and ignoring categorical labels.

Can you try exporting with X-Y coordinates and seeing if the issue persists?

**Comment by hkuyam008** [comment]
Exporting with X-Y coordinates does work. however when I used the labelbox2coco script to convert the Labelbox XY exported format to COCO format, the information about categories is lost during conversion and the coco format just has info regarding supercategorey

"categories": [{
		"supercategory": "component",
		"id": 1,
		"name": "component"
	}]

There is no info regarding my 2 categories "button" & "dropdown" in the coco format json.
I have attached both the Labelbox XY & converted coco formats for reference

[exported & converted data.zip](https://github.com/Labelbox/Labelbox/files/2391902/exported.converted.data.zip)

**Comment by feynmanliang** [comment]
I see. I'll add this feature (COCO export should preserve category labels) to the engineering backlog. Can you provide some details of what you expect the output to look like?
